qtfred/src/main.cpp

```diff
@@ -180,12 +182,14 @@ int main(int argc, char* argv[]) {
 	// Use this to keep the app responsive
 	game_busy_callback(game_busy_callback);
 
+	// Show all top level windows that are our window
```

Comment:
You should fix this comment. It used to be true but this PR changes the following loop to only show the *first* top level window. It shouldn't cause any noticable changes, though, since we only ever have one `FredView`.